integrations/tensorflow/e2e/README.md

```diff
@@ -23,6 +23,56 @@ targets with `--target_backends=tf,iree_vmla,iree_llvmjit` (that is, by omitting
 The test suites can be run excluding Vulkan by specifying
 `--test_tag_filters="-driver=vulkan"` in the `bazel test` invocation.
 
+
+## Compiling `tf.Module`s
+
+Compatible TensorFlow modules can be compiled to specific IREE backends using
+`tf_utils.compile_tf_module`. This also optionally saves compilation artifacts
+to a specified directory. These artifacts include: MLIR across various
+lowerings, a SavedModel, and the compiled VM FlatBuffer.
+
+A minimal example of this is the following:
+
+```python
```

Comment:
I'd rather have this large block of code in an "examples" directory or something. Without having it run it is fairly likely to drift.

Reply:
Deleted.